Microsoft.Toolkit.Uwp.UI.Controls/BladeControl/BladeControl.xaml

```diff
@@ -59,14 +78,14 @@
                                        VerticalAlignment="Center"
                                        Foreground="{TemplateBinding TitleBarForeground}"
                                        Text="{TemplateBinding Title}" />
-                            <Button Name="CloseButton"
-                                    Grid.Column="1"
+                            <Button Grid.Column="1"
```

Comment:
None of these properties in this control have actually changed, just been moved around. Not needed.